src/test/java/bio/terra/service/snapshot/SnapshotDaoTest.java

```diff
@@ -72,7 +73,12 @@ public void setup() throws Exception {
             .defaultProfileId(profileId.toString());
         dataset = DatasetUtils.convertRequestWithGeneratedNames(datasetRequest);
         String createFlightId = UUID.randomUUID().toString();
-        datasetId = datasetDao.createAndLock(dataset, createFlightId);
+        datasetId = UUID.randomUUID();
+        Instant createdDate = Instant.now();
+        dataset
+            .id(datasetId)
+            .createdDate(createdDate);
```

Comment:
`createdDate` is being passed in to `DatasetDao.createAndLock()` but it's currently unused in that method (unlike `id`, which is used), is this a change that should be rolled back?